docs/startup/code-examples/versioned/index.md

```diff
@@ -47,3 +47,67 @@ In addition to the cookbook, you can also consult these:
 
 
 - [Amazon Kinesis](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/kinesis) and [Azure Eventhubs](https://github.com/apache/samza-hello-samza/tree/latest/src/main/java/samza/examples/azure) examples that cover how to consume input data from the respective systems.
+
+### Apache Beam API examples
+
+The easiest way to get a copy of the WordCount examples in Beam API is to use the following command:
```

Comment:
a kind reminder to users that they need to install maven.  We could mention the following example has been tested on both Linux and Mac